### PR TITLE
windows if label

### PR DESCRIPTION
**Pull request by romanovvlad**
Trying to trigger windows build and testing if a specific comment is posted.
The are complications:
1. For some reason, a workflow triggered by a comment is not displayed in the PR status(where all github actions are displayed). So, trying to update it manually.
2. The github hash provided for the default branch rather than for the PR branch.

### CODE REVIEW

**Comment by romanovvlad** [comment]
windu please

**Comment by bader** [comment]
> windu please

okna please ;)

**Comment by romanovvlad** [comment]
windu please

**Comment by romanovvlad** [comment]
windu please

**Comment by Fznamznon** [comment]
> too much of zeleniy tea?

I thought that was romashka.

**Comment by dm-vodopyanov** [comment]
One more raspechatka on the cubicle wall? :)

File: .github/workflows/windows_test_comment_trigger.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
+name: Test windows on the comment
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  windows_test_preparation:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.issue.pull_request && github.event.comment.body == '/windu please' }}
```
Comment (vvkalinin24) [inline .github/workflows/windows_test_comment_trigger.yml:10]: Could we use some readable words? Like: "Windows please" or "Windows"

File: .github/workflows/windows_test_comment_trigger.yml [hunk 2]
```diff
@@ -0,0 +1,87 @@
+name: Test windows on the comment
+
+on:
+  issue_comment:
```
Comment (pvchupin) [inline .github/workflows/windows_test_comment_trigger.yml:4]: Reading through docs it seems like issue_comment is probably the only option, it's really strange it doesn't have PR sha. At the same time all pull_request* actions have PR sha. Have you actually tried this one for example? https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_review
I'm not sure what this statement really means: 
```
A pull request review is a group of pull request review comments in addition to a body comment and a state
```

File: .github/workflows/windows_test_comment_trigger.yml (hunk 1 above)
Comment (romanovvlad) [reply .github/workflows/windows_test_comment_trigger.yml:10]: Done.

File: .github/workflows/windows_test_comment_trigger.yml (hunk 2 above)
Comment (romanovvlad) [reply .github/workflows/windows_test_comment_trigger.yml:4]: This event triggers the workflow when a review comment is added or edited. Review comment != pull request comment.

**Comment by romanovvlad** [comment]
Merging per agreement with @pvchupin. Until it's merged it cannot be really verified since the trigger on event works for workflows merged in the default(sycl for inte/llvm) branch.

File: .github/workflows/windows_test_comment_trigger.yml [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+        SHA: ${{ steps.sha.outputs.result }}
+
+  # windows_default:
+  #   runs-on: ubuntu-latest
+  #   needs: [windows_test_preparation]
+  #   steps:
+  #   - name: print_ref
+  #     run: |
+  #       echo "REF == = = = = = = ${{github.ref}}"
+  #       echo "REF == = = = = = = ${{ needs.windows_test_preparation.outputs.PR_SHA }}"
```
Comment (bader) [inline .github/workflows/windows_test_comment_trigger.yml:87]: @romanovvlad, can we remove this commented out part of the script?

Comment (romanovvlad) [reply .github/workflows/windows_test_comment_trigger.yml:87]: Yes. It was done here: https://github.com/intel/llvm/commit/017bcdcfbaf3c400ae67e7cf4ca135e623c5ecc6

**Pull request merged**